### PR TITLE
Purge Cloudflare via GHA

### PR DESCRIPTION
**Pull request by delphiactual**
This should work as-is but I would like to make changes directly in CloudFlare, such that app, beta, and website have different zones.

### CODE REVIEW

File: .github/workflows/deploy-beta.yml [hunk 1]
```diff
@@ -73,12 +73,13 @@ jobs:
...
-      - name: Purge CloudFlare cache
-        run: ./build/purge-cloudflare.sh
+      - name: Cloudflare Purge Cache
+        uses: jakejarvis/cloudflare-purge-action@v0.3.0
         env:
+          CLOUDFLARE_ZONE: ${{ secrets.CLOUDFLARE_ZONE }}
           CLOUDFLARE_KEY: ${{ secrets.CLOUDFLARE_KEY }}
           CLOUDFLARE_EMAIL: ${{ secrets.CLOUDFLARE_EMAIL }}
-          APP_DOMAIN: beta.destinyitemmanager.com
+          PURGE_URLS: '["https://beta.destinyitemmanager.com", "https://beta.destinyitemmanager.com/index.html", "https://beta.destinyitemmanager.com/version.json", "https://beta.destinyitemmanager.com/service-worker.js", "https://beta.destinyitemmanager.com/return.html", "https://beta.destinyitemmanager.com/.well-known/assetlinks.json", "https://beta.destinyitemmanager.com/.well-known/apple-app-site-association", "https://beta.destinyitemmanager.com/manifest-webapp-6-2018.json", "https://beta.destinyitemmanager.com/manifest-webapp-6-2018-ios.json"]'
```
Comment (bhollis) [inline .github/workflows/deploy-beta.yml:82]: The thing I liked about our custom script is that we could maintain the list of URLs in one place for both domains.

**Review by bhollis — COMMENTED** [review]
I *think* multiple zones costs money, but I'm not sure. Only @SunburnedGoose can add people to CloudFlare.

File: .github/workflows/deploy-beta.yml (hunk 1 above)
Comment (delphiactual) [reply .github/workflows/deploy-beta.yml:82]: Once I get them all put together I can consolidate them, https://docs.github.com/en/actions/using-workflows/reusing-workflows

Comment (delphiactual) [reply .github/workflows/deploy-beta.yml:82]: also adding a flavor environment variable to assist beforehand

Comment (delphiactual) [reply .github/workflows/deploy-beta.yml]: https://github.com/DestinyItemManager/d2-additional-info/runs/5525233093?check_suite_focus=true

environment variable substitution working

**Pull request closed without merging**